### PR TITLE
MP v2 codemods

### PR DESCRIPTION
**Pull request by alloy**
- [x] `update-import-source`: a codemod that makes it easy to update imports from e.g. `schema/object_identifaction` to `schemaV2/object_identifaction`
- [x] `update-graphql-field-definition-case`: a codemod that updates `snake_case` to `camelCase`
  - [x] Output fields
  - [x] Output field arguments
  - [x] Input fields
  - [x] Input field arguments
- [x] `remove-deprecated-graphql-field-definitions`: a codemod that removes field definitions that have a `deprecationReason`:
  - [x] Output fields
  - [x] Enum values

### CODE REVIEW

File: src/update-graphql-field-definition-case.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      case "usd":
+        return "USD"
+      case "utc":
+        return "UTC"
+      case "md":
+        return "MD"
+      default:
+        return c
+    }
+  })
```
Comment (alloy) [inline src/update-graphql-field-definition-case.ts]: Any others?

Comment (alloy) [reply src/update-graphql-field-definition-case.ts]: cc @ashkan18

File: src/update-graphql-field-definition-case.ts [hunk 2]
```diff
@@ -0,0 +1,297 @@
+/**
+ * First remove deprecated fields from the schema, as there will likely be some
```
Comment (ashkan18) [inline src/update-graphql-field-definition-case.ts]: is this comment correct?

File: src/update-graphql-field-definition-case.ts (hunk 1 above)
Comment (ashkan18) [reply src/update-graphql-field-definition-case.ts]: hmmm... i think thats it, i thought originally `HREF` maybe but looks like we do `Href` which seems better to me.

Comment (alloy) [reply src/update-graphql-field-definition-case.ts]: Ah yes, hmm, there are original/official docs that _do_ capitalise when they do not speak about the `a` attribute but the concept, so maybe we should do so after all. I.e. all abbreviations/acronyms etc are capitalised?

E.g.

> HREF is an abbreviation for "Hypertext REFerence"

From https://en.wikipedia.org/wiki/Hyperlink

File: src/update-graphql-field-definition-case.ts (hunk 2 above)
Comment (alloy) [reply src/update-graphql-field-definition-case.ts]: I think so, yes. What makes you ask?

File: scripts/remove-blank-lines-from-unstaged-changes [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# recast has an issue where blank lines get added around new properties https://github.com/benjamn/recast/issues/370
```
Comment (zephraph) [inline scripts/remove-blank-lines-from-unstaged-changes]: Oh, clever! 👍

**Comment by alloy** [comment]
Ok, with this review already reviewed I'll go ahead and merge this as-is. Any other MP codemods will have to be follow-up PRs.

**Pull request merged**